taskcluster/docker/linux/Dockerfile

```diff
@@ -29,6 +29,8 @@ ENV SHELL=/bin/bash \
     HOME=/builds/worker \
     PATH=/builds/worker/.local/bin:/root/.local/bin:$PATH
 
+ENV MOZ_LOG=HTMLMediaElement:4,HTMLMediaElementEvents:4,cubeb:5,PlatformDecoderModule:5,AudioSink:5,AudioSinkWrapper:5,MediaDecoderStateMachine:4,MediaDecoder:4,MediaFormatReader:5,GMP:5,EME:5
```

Comment:
#YOLO

(but also, maybe a comment w/ link to this arcane magic of CPP logging -- or whatever the very specific flags are?)

Reply:
Oh no I didn't mean to leave this in there!